lua/lspconfig/server_configurations/sourcery.lua

```diff
@@ -0,0 +1,49 @@
+local util = require 'lspconfig/util'
+
+local root_files = {
+  'pyproject.toml',
+  'setup.py',
+  'setup.cfg',
+  'requirements.txt',
+  'Pipfile',
+  'pyrightconfig.json',
+}
+
+return {
+  default_config = {
+    cmd = { 'sourcery', 'lsp' },
+    filetypes = { 'python' },
+    init_options = {
+      editor_version = 'vim',
+      extension_version = 'vim.lsp',
+      token = '<YOUR_TOKEN>',
```

Comment:
Make token nil, add a default `on_new_config` that asserts token is not nil and throws an error message with notify_once from core.